python/lsst/pipe/tasks/mergeDetections.py

```diff
@@ -230,8 +230,8 @@ def __init__(self, butler=None, schema=None, initInputs=None, **kwargs):
         self.makeSubtask("skyObjects")
         self.schema = self.getInputSchema(butler=butler, schema=schema)
 
-        filterNames = [getShortFilterName(name) for name in self.config.priorityList]
-        filterNames += [self.config.skyFilterName]
+        filterNames = list(self.config.priorityList)
+        filterNames.append(self.config.skyFilterName)
```

Comment:
What do the configs for `priorityList` look like in practice? Do they use bands to begin with?

Reply:
It was physical in HSC, but band in CFHT and DECam. @TallJimbo stated on slack "I believe in Gen3 the priorityList is already band, and has to be, because coadds always use band."

I am changing it to `band` and updating obs_subaru's priorityList entries to reflect this.